README.md

```diff
@@ -14,7 +14,15 @@ Please [file an issue](https://github.com/odk-x/tool-suite-X/issues) if you can'
 
 There are two options for building and viewing ODK-X docs locally: [using Docker](#docker) or [setting up a local Python/Sphinx environment](#python-environment). We generally recommend starting with the Docker image unless you already have a Sphinx environment set up. The [Contributor Guide](https://docs.odk-x.org/contributing) describes the philosophy behind the docs, style considerations, how to write restructured text and more.
 
-## <a name="docker"></a>Using Docker
+There are two options to build ODK-X documents 
+
+* [Docker Hosted Build Enviornment](#docker)
+* [Python Build Enviornment](#python-environment)
+
+The Docker recommended enviornment is recommended because of fewer setup steps required and less enviornmental variable paths that need to be set.
```

Comment:
recommended environment is recommended is probably redundant, switch to "The Docker environment is recommended because... "